packages/frontend/src/components/SimpleStatistic/SimpleStatistics.styles.tsx

```diff
@@ -8,27 +8,25 @@ export const SimpleStatisticsWrapper = styled.div`
 `;
 
 export const BigNumberContainer = styled.div`
-    border: 1px solid ${Colors.GRAY4};
-    border-radius: 0.5em;
-    box-shadow: -0.375em 0.5em 0.313em -0.25em rgba(0, 0, 0, 0.06);
-    max-width: fit-content;
-    padding: 1.875em 2.5em;
     position: absolute;
     top: 50%;
     left: 50%;
+    text-align: center;
     transform: translate(-50%, -50%);
 `;
 
 export const BigNumber = styled.div`
-    font-size: 2.625em;
+    font-size: 4em;
+    line-height: 1.196em;
     font-weight: 500;
-    color: ${Colors.DARK_GRAY5};
+    color: ${Colors.DARK_GRAY4};
 `;
 
 export const BigNumberLabel = styled.h2`
     text-align: center;
-    color: ${Colors.GRAY3};
-    font-weight: normal;
-    font-size: 1em;
+    color: #979797;
```

Comment:
@nathaliabj @PriPatel is there a blueprint colour we could use for the label text ?

Reply:
@ZeRego @nathaliabj 
I did not mean to change the colour - I have no idea why its changed in my file - apologies. I think the original was **@gray3** and thats great.